packages/web/integrations/notifi/notifi-context.tsx

```diff
@@ -0,0 +1,106 @@
+import { WalletAccount } from "@cosmos-kit/core";
+import { NotifiParams } from "@notifi-network/notifi-react-card";
+import { observer } from "mobx-react-lite";
+import dynamic from "next/dynamic";
+import React, {
+  FunctionComponent,
+  PropsWithChildren,
+  useEffect,
+  useState,
+} from "react";
+
+import { useStore } from "../../stores";
+import { NotifiConfigContext } from "./notifi-config-context";
+import { NotifiModalContextProvider } from "./notifi-modal-context";
+
+const NotifiContext = dynamic<NotifiParams>(
+  () =>
+    import("@notifi-network/notifi-react-card").then(
+      (mod) => mod.NotifiContext
+    ),
+  {
+    ssr: false,
+  }
+);
+
+interface RequiredInfo {
+  account: WalletAccount;
+  chainId: string;
+}
+
+const NotifiContextProviderImpl: FunctionComponent<PropsWithChildren<{}>> =
+  observer(({ children }) => {
+    const {
+      chainStore: {
+        osmosis: { chainId },
+      },
+      accountStore,
+    } = useStore();
+    const [info, setInfo] = useState<RequiredInfo | undefined>();
+    const wallet = accountStore.getWallet(chainId);
+    const walletConnected = Boolean(wallet?.isWalletConnected);
+
+    useEffect(() => {
+      if (!walletConnected) {
+        return;
+      }
+      getRequiredInfo(chainId, accountStore).then((info) => setInfo(info));
+    }, [accountStore, chainId, wallet, walletConnected]);
+
+    if (info === undefined) {
+      return <>{children}</>;
+    }
+
+    return (
+      <NotifiContext
+        env="Development"
+        walletBlockchain="OSMOSIS"
+        dappAddress="osmosis"
+        accountAddress={info.account.address}
+        walletPublicKey={Buffer.from(info.account.pubkey).toString("base64")}
+        signMessage={async (message: Uint8Array): Promise<Uint8Array> => {
+          const wallet = accountStore.getWallet(info.chainId);
+          if (
+            wallet === undefined ||
+            wallet.client.signArbitrary === undefined
+          ) {
+            throw new Error("Wallet does not support signing Notifi messages");
+          }
+          const result = await wallet.client.signArbitrary(
+            info.chainId,
+            info.account.address,
+            message
+          );
+          return Buffer.from(result.signature, "base64");
+        }}
+      >
+        <NotifiConfigContext
+          type="SUBSCRIPTION_CARD"
+          id="c5fb30811f5b47b79dd9a400480c2670"
```

Comment:
Do you guys have a testnet environment? If so, let's use IS_TESTNET to use that. If you don't have an env, I think you should for testing.

Reply:
Is it ok for me to add new variables in .env . Like below
```
NEXT_PUBLIC_NOTIFI_CARD_ID_TESTNET=xxx
NEXT_PUBLIC_NOTIFI_CARD_ID_PRODUCTION=xxx
```
or you will prefer put the id in-line.

Reply:
For now the updated PR is using in-line. can change to env anytime.

Reply:
I think inline is fine for now